README.md

```diff
@@ -0,0 +1,212 @@
+
+- [Tesla Booking Frontend Kanban Board](https://github.com/users/ITurres/projects/6/views/1)
+<img src="./Backend.png">
+- [Tesla Booking Backend Kanban Board](https://github.com/users/mahammad-mostafa/projects/4)
+<img src="./Frontend.png">
+
+**Team Members**
+- Demes Abera Ameneshoa - GitHub: [@demesameneshoa](https://github.com/demesameneshoa)
+- Fatema Nazari [@78fatemanazari](https://github.com/78fatemanazari)
+- Arthur Emanuel G. Iturres [@ITurres](https://github.com/ITurres)
+- Mahammad Mostafa [@mahammad-mostafa](https://github.com/mahammad-mostafa)
+
+
+**ER Diagram**
+The  API is designed as per the ER Diagram shown below which dictates the association of models and their relationships.
+
+<img src="./ER Diagram.png">
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- [x] **Ruby on Rails As an API**
+- [x] **Postgres Database**
+- [x] **API Routed to Provide Response for Requests**
+- [x] **Git Version Control**
+- [x] **User Authentication Managed by Devise**
+- [x] **JWT Token as Authorization Key**
+- [x] **Models Associations**
+- [x] **API Documentation**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Documentation <a name="documentation"></a>
+
+- You can view the input json and response json formats and also   you can interact with the API database to visualize the input and output flow of requests and responses.
+- [Api docs](https://tesla-booking-api-x2xe.onrender.com/api-docs)
+
+## Live Demo <a name="live-demo"></a>
+- [API URL](https://tesla-booking-api-x2xe.onrender.com)
+
+## 🎨 Link to Frontend <a name="link-to-frontend"></a>
+
+- [Frontend](https://github.com/ITurres/tesla-booking-front-end)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ mkdir tesla-booking-api
+ cd tesla-booking-api
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/mahammad-mostafa/tesla-booking-back-end
+```
+
+### Database setup
+
+Connect to your local postgres database:
+
+```sh
+  Inside the config/database.yml file add your
+
+  POSTGRES_USER=<username>
+  POSTGRES_PASSWORD=<password> 
+ 
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd tesla-booking-back-end
+  bundle install
+  rails db:create - create the database
```

Comment:
- [ ] When I try to create the database, according to the instructions, I get this:

![image](https://github.com/mahammad-mostafa/tesla-booking-back-end/assets/57421823/208012ca-2430-43d8-96f1-e8a13de079fd)

This is because your users will need to create their own master.key.

These are the steps:

```
1. Remove config/master.key and config/credentials.yml.enc if they exist.
2. Run in the terminal: EDITOR=code rails credentials:edit 
3. Close the editor that opens.
4. This command will create a new master.key and credentials.yml.enc if they do not exist.
```

Result:

![image](https://github.com/mahammad-mostafa/tesla-booking-back-end/assets/57421823/5abdfd23-d024-4c17-9f48-0ecf97fb457d)


Kindly, reflect this in your instructions so your users are able to setup the database correctly. 👍